crates/ln-dlc-node-tests/src/lib.rs

```diff
@@ -0,0 +1,2 @@
+#[cfg(test)]
+mod tests;
```

Comment:
It would be interesting to benchmark the differences between the original, this version and this version but with a single integration test crate i.e.

```
tests/
  integration/
    main.rs
    foo.rs
    bar.rs
```

I always go back to this blog post when thinking about this topic: https://matklad.github.io/2021/02/27/delete-cargo-integration-tests.html